game.c

```diff
@@ -15,155 +15,167 @@
 /*todo implement collision detection
  */
 
-struct argument_parameters {
-};
+struct argument_parameters {};
 
 struct frame_time_info {
-	struct timespec beginning_of_framrate_calc;
-	struct timespec prev_time;
-	double frame_rate_target;
-	long ns_per_frame;
-	int num_updates;
-	int current_framerate;
+    struct timespec prev_time;
+    double frame_rate_target;
+    long ns_per_frame;
+    int num_updates;
+    int current_framerate;
 };
 
 void init_frame_rate_parameters(struct frame_time_info *frames_data) {
-	frames_data->frame_rate_target = 10;
-	frames_data->ns_per_frame = 1000000000 / frames_data->frame_rate_target;
+    frames_data->frame_rate_target = 60;
+    frames_data->ns_per_frame = 1000000000 / frames_data->frame_rate_target;
 }
 
-void init_SDL_settings() {
+void init_SDL_settings() {}
+
+int init_arguments() { return 0; }
+
+int process_arguments(int ac, char **av,
+                      struct argument_parameters *arguments) {
+    return 0;
 }
 
-int init_arguments() {
-	return 0;
+long ONE_NANOSECOND = 1000000000;
```

Comment:
This is horrible / shouldn't be defined here

Reply:
putting this in but as ONE_SEC_AS_NANO in a #define

Reply:
Nice, that's way clearer / better way to handle.

In Swift we also have a way to make the number clearer. We do 1_000_000_000. Not sure if C has the same thing

Reply:
I guess in the Objective-C days we did 1 * 1000 * 1000 * 1000 lol